Spring-Boot/Spring-Boot-Exam/src/test/java/tw/waterball/judgegirl/springboot/exam/controllers/ExamControllerIT.java

```diff
@@ -37,7 +38,8 @@
     JpaExamDataPort examRepository;
     @Autowired
     JpaExamParticipationDataPort examParticipationRepository;
-
+    @Autowired
+    JpaQuestionDataPort questionRepository;

```

Comment:
I missed this one, don't directly inject the `JpaXxxDataPort`
Use `QuestionRepository` instead of `JpaQuestionDataPort`, ... and so on